python-client/giskard/models/automodel/__init__.py

```diff
@@ -0,0 +1,30 @@
+from importlib import import_module
+
+
+class AutoModel:
+    def __new__(cls, *args, **kw):
```

Comment:
I have some reservations on the design of this. In my opinion it is a bit tricky to have a fake class `AutoModel` which can never be instantiated, leading to things like:
```py
model = AutoModel(my_classifier)
assert isinstance(model, AutoModel)  # fail!
```
This would be counterintuitive for me.

Maybe we should just put this code in the parent `Model` class.

Actually, I would rather go for a standard [factory pattern](https://en.wikipedia.org/wiki/Factory_method_pattern), like Huggingface does.

Something like
```py
from giskard import Model

model = Model.from_classifier(my_classifier)
```
?

Reply:
done